### DISCUSSION TITLE
Item: Secret Scroll

### DISCUSSION THREAD

**Discussion by ntseng** [Imaginary-Horizons-Productions/Prophets-of-the-Labyrinth#101] [Ideas]
Has a different effect based on the archetype of the user:
Assassin - 
Detective - 
Chemist -
Knight - 
Hemomancer - 
Legionnaire - 
Martial Artist - 
Ritualist -
Beast Tamer - Use your Pet's second move

**Comment by doubledh** [comment]
should they all be thematically creative and/or directly synergistic?

**Reply by ntseng** [reply]
The effect is intended to be thematic to the archetype (so it's memorable), fairly powerful (so it gets engaged with), and situational (to amplify the "who should use the scroll?" decision).
